atlp-rwanda/rca-barefoot-team3-fe
# feat rooms list and Book room functionality

### What does this PR do?
This PR fetch rooms and allow user to book a room

### Steps required to achove this task
1. Fetch all rooms and their details
2. Get current user id
3. Implement form for checking and checkout dates
3. Since we have rooms and their details use current user id and room id to book a room

### How to test the functionality
1. Make sure you are logged in
2. go to dashboard and click on rooms 
3. Click on add Book button

### Screenshots
rooms list
![image](https://user-images.githubusercontent.com/76569213/236766960-2f523547-f081-4772-88ff-fd8504a30614.png)

Booking Modal

![image](https://user-images.githubusercontent.com/76569213/236767128-db63144a-cace-4284-a5eb-6953cbb6a352.png)
